EveHQ.Prism/Classes/StructurePreference.vb

```diff
@@ -0,0 +1,113 @@
+﻿Imports EveHQ.EveData
+
+''' <summary>
+''' A Structure Definition
+''' </summary>
+Public Class StructurePreference
```

Comment:
May be just Structure but it's a bit misleading because it's so common term. The is up to you.